npm-shrinkwrap.json

```diff
@@ -2,38 +2,1193 @@
   "name": "convict",
   "version": "1.3.0",
   "dependencies": {
+    "abbrev": {
```

Comment:
This is a lot of shrinkwrap diffs!
I think there is a shrinkwrap script in the [package.json](https://github.com/mozilla/node-convict/blob/master/package.json#L34) which you can use to only add the production dependencies.
This might also be a npm@2 vs npm@3 thing, since they handle shrinkwrap differently. I'm not sure what version of npm @madarche usually uses.